mod/01_filter/set/right/filter_search.rb

```diff
@@ -15,7 +15,11 @@ def params_keys
   []
 end
```

Comment:
this is where I get confused.  usually params_keys = default + advanced, right?  Why is it different here?